src/hooks/useCommand.js

```diff
@@ -1,35 +1,55 @@
+import MANUAL from "../data/manual";
 import useTerminal from "./useTerminal";
+import useMissions from "./useMissions";
+import useFlag from "./useFlag";
 
-// TODO: Implement commands
-export const commands = {
-  help: (argsList) => "Help command output.",
-  list: (argsList) => "List command output.",
-  select: (argsList) => "Select command output.",
-  status: (argsList) => "Status command output.",
-  flag: (argsList) => "List command output.",
-  clear: (argsList) => "Clear command output.",
-};
+const useCommand = () => {
+  const { stdIn, stdOut, clear } = useTerminal();
+  const { getMissions, selectMission, currentMission, getStatus } =
+    useMissions();
+  const { checkFlag } = useFlag(currentMission);
+
+  const handleHelp = () => stdOut(MANUAL);
+  const handleClear = () => clear();
+  const handleList = () => stdOut(getMissions());
+  const handleStatus = () => stdOut(getStatus());
+  const handleSelect = (args) => {
```

Comment:
**AI comment for this func. Of course is up to you ;)**

_The handleSelect function could be simplified by using a default value for the mission ID parameter instead of checking for NaN. For example:_ `const handleSelect = (args) => stdOut(selectMission(parseInt(args[0], 10)));`.

Reply:
Nie chcemy default value